bootstrap/backends/ldap/ldap.go

```diff
@@ -87,9 +91,24 @@ func NewIdentityManager(bs bootstrap.Bootstrap) (identity.Manager, error) {
 		subMapping = strings.Split(subMappingString, " ")
 	}
 
+	// Use a clone here to avoid changing the config of other possible users of the config.
+	tlsConfig := config.TLSClientConfig.Clone()
+	if caCertFile := os.Getenv("LDAP_TLS_CACERT"); caCertFile != "" {
+		if pemBytes, err := ioutil.ReadFile(caCertFile); err == nil {
+			rpool, _ := x509.SystemCertPool()
+			if rpool.AppendCertsFromPEM(pemBytes) {
+				tlsConfig.RootCAs = rpool
+			} else {
+				return nil, fmt.Errorf("Error appending CA certificate(s) '%s' to pool", caCertFile)
+			}
+		} else {
+			return nil, errors.Wrapf(err, "Error reading CA certificate '%s'", caCertFile)
```

Comment:
no need to use the `errors` package. Instead use error wrapping via the [fmt package](https://pkg.go.dev/fmt#Errorf) (since Go 1.13, lico requires Go 1.16).